rpcs3/ds3_pad_handler.cpp

```diff
@@ -87,13 +87,13 @@ bool ds3_pad_handler::Init()
 
 	if (warn_about_drivers)
 	{
-		LOG_ERROR(HLE, "[DS3] One or more DS3 pads were detected but couldn't be handled because of drivers");
+		LOG_ERROR(HLE, "[DS3] One or more DS3 pads compatible were detected but couldn't be handled because of drivers");
 		LOG_ERROR(HLE, "[DS3] We recommend you use Zadig( https://zadig.akeo.ie/ ) to change your ds3 drivers to WinUSB ones");
 	}
 	else if (controllers.size() == 0)
-		LOG_WARNING(HLE, "[DS3] No controllers found!");
+		LOG_WARNING(HLE, "[DS3] No controllers compatible found!");
 	else
-		LOG_SUCCESS(HLE, "[DS3] Controllers found: %d", controllers.size());
+		LOG_SUCCESS(HLE, "[DS3] Controllers compatible found: %d", controllers.size());
```

Comment:
```suggestion
		LOG_SUCCESS(HLE, "[DS3] Compatible controllers found: %d", controllers.size());
```
Also "%d compatible controlller(s) found" would work.